spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-k8s-client-discovery-server/src/test/java/org/springframework/cloud/kubernetes/discoveryclient/it/DiscoveryClientFilterNamespaceDelegate.java

```diff
@@ -17,14 +17,18 @@
 package org.springframework.cloud.kubernetes.discoveryclient.it;
 
 import java.time.Duration;
+import java.util.LinkedHashMap;
```

Comment:
piggy back some assertions on an existing integration test. It makes sure:

- discovery server properly sees the heart beat event coming from reactive discovery k8s client (via logs assertion)
- discovery client properly invokes the new endpoint in the discovery server and publishes a heartbeat (via logs assertions + a new GET endpoint in the new test controller)